src/core/wb.js

```diff
@@ -255,6 +255,16 @@ var getUrlParts = function( url ) {
 			"#999999"
 		],
 
+		// Get and generate a unique session id
```

Comment:
What is the issue, exception of how it was packaged? That insertion is already in a separate commit https://github.com/wet-boew/wet-boew/pull/7982/commits/a42b4cd3ef39f345bbeb2138ddf3ca0beddbed42